src/org/zaproxy/zap/extension/ascanrulesBeta/HPP.java

```diff
@@ -313,14 +313,13 @@ public int getRisk() {
 
 	@Override
 	public int getCweId() {
-		return 352;
+		return 20; //CWE-20: Improper Input Validation
```

Comment:
I think we should mention somewhere that the IDs were changed, in case someone is relying on those values.